Applications/MIDIMonitor/SMMAppController.m

```diff
@@ -332,9 +331,21 @@ - (void)failedToConnectToSpyClient
 
 - (void)sourceEndpointsAppeared:(NSNotification *)notification
 {
-    if ([[NSUserDefaults standardUserDefaults] boolForKey:SMMOpenWindowsForNewSourcesPreferenceKey]) {
+    NSInteger option = [[NSUserDefaults standardUserDefaults] integerForKey:SMMAutoConnectNewSourcesPreferenceKey];
```

Comment:
It would be nice if there were a migration path from the old default to the new one. As it is now, you're completely ignoring the old preference, which is a bit rude to our users.

You could, for instance, add code to `applicationWillFinishLaunching:` that reads the old default and sets the corresponding value for the new default. (There is no need to remove the old default afterwards -- it's nice to leave it around in case someone runs an older version of the app.)